<launch>
    <arg name="map_location" default="$(find st_map_function)/map/xy_save.pcd" />
    <node pkg="pointcloud_registration" type="show_map" name="show_map" >
		<param name="map_location" value="$(arg map_location)" />
		<param name="global_frame_id" value="map" />
	</node>
</launch>
